assets/wire/actions.ts

```diff
@@ -506,6 +507,36 @@ export function reloadMyTopics(reloadTopic: any = false) {
     };
 }
 
+export const UPDATE_FOLDERS = 'UPDATE_FOLDERS';
+
+/**
+ * @param {bool} global - fetch company or user folders
+ * @param {bool} skipDispatch - if true it won't replace folders in store
+ */
+export function fetchFoldersWire() {
+    return (dispatch: any, getState: any) => {
+        const state = getState();
+        const companyTopicsUrl = getFoldersUrl(state, true);
+        const userTopicsUrl = getFoldersUrl(state, false);
+
+        return Promise.all([
+            state.company ? server.get(companyTopicsUrl).then(({_items}: {_items: Array<any>}) => _items) : Promise.resolve([]),
+            server.get(userTopicsUrl).then(({_items}: {_items: Array<any>}) => _items),
+        ]).then(([companyFolders, userFolders]) => {
+            dispatch({
+                type: UPDATE_FOLDERS,
+                payload: {
+                    companyFolders: companyFolders,
+                    userFolders: userFolders,
+                },
+            });
+        }).catch((error) => {
```

Comment:
is there a point to catch it at all here if the only thing you're doing is printing it to the console? Wouldn't it log automatically if promise fails?